usr/src/uts/common/fs/zfs/arc.c

```diff
@@ -1080,6 +1080,7 @@ typedef struct l2arc_read_callback {
 	blkptr_t		l2rcb_bp;		/* original blkptr */
 	zbookmark_phys_t	l2rcb_zb;		/* original bookmark */
 	int			l2rcb_flags;		/* original flags */
+	void			*l2rcb_data;		/* temporary buffer */
```

Comment:
Are you sure this PR is complete? I can't seem to find where l2rcb_data is being populated.